assets/store_descriptions/generate_and_validate.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+# Exit on error
+set -e
+
+# Remove old descriptions
+find fastlane/metadata/android/ -name "*_description.txt" -delete
```

Comment:
You are expecting the script to be executed from the relative repository root.
You should definitely check on that first. Either by working [relative to the script location](https://stackoverflow.com/questions/59895/getting-the-source-directory-of-a-bash-script-from-within) or by ensuring [execution inside the repository root](https://stackoverflow.com/questions/59838/check-if-a-directory-exists-in-a-shell-script)

Reply:
I think adding a comment is enough. Ideally the script is run by travis only.

Reply:
Instead of a comment how about this as command before this one.

```
[ -d "fastlane/metadata/android" ] || exit 1
```